xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/informationinline.vm

```diff
@@ -44,146 +237,23 @@ $xwiki.jsfx.use('uicomponents/edit/editableProperty.js', {'forceSkinAction': tru
-        </dt>
-        #set ($displayParams = {'value': $tdoc.realLocale})
-        <dd data-key="locale">#displayProperty_locale($displayParams)</dd>
-        ##
-        ## Original Locale
-        ##
-        #if ($tdoc.realLocale != $doc.realLocale)
-          <dt>
-            <label>$escapetool.xml($services.localization.render('core.viewers.information.originalLocale'))</label>
-          </dt>
-          <dd data-key="originalLocale">#docTranslationLink($doc.realLocale 'view')</dd>
-        #end
-        ##
-        ## Translations
-        ##
-        #set ($existingTranslations = $doc.translationLocales)
-        ## Show the list of translations if multilingual is on and either the original page is translatable (has a
-        ## locale specified) or it has existing translations.
-        #if ($xwiki.isMultiLingual() && ("$!doc.realLocale" != '' || !$existingTranslations.isEmpty()))
-          #set ($supportedLocales = $xwiki.availableLocales)
-          ## Remove the translations that we already display above.
-          #set ($discard = $supportedLocales.remove($tdoc.realLocale))
-          #set ($discard = $supportedLocales.remove($doc.realLocale))
-          #if (!$supportedLocales.isEmpty())
-            <dt>
-              <label>$escapetool.xml($services.localization.render('core.viewers.information.translations'))</label>
-            </dt>
-            #set ($output = [])
-            #foreach ($supportedLocale in $supportedLocales)
-              #if ($existingTranslations.contains($supportedLocale))
-                #set ($discard = $output.add($stringtool.trim("#docTranslationLink($supportedLocale 'view')")))
-              #elseif ($hasEdit)
-                ## Show the link to create the missing translation if the current user can edit the page. Use the
-                ## preferred edit mode (in-place versus stand-alone) depending on the user preferences, in order to
-                ## replicate the behavior of the Translate button.
-                #set ($action = 'edit')
-                #set ($fragment = $NULL)
-                #getDefaultDocumentEditor($defaultEditMode)
-                #if ($services.edit.document.inPlaceEditingEnabled() && $defaultEditMode == 'wysiwyg')
-                  ## Link to the in-place page editor which currently uses the view action.
-                  #set ($action = 'view')
-                  #set ($fragment = 'translate')
-                #end
-                #set ($discard = $output.add(
-                  "<span class='wikicreatelink skipCreatePagePopup'>" +
-                    $stringtool.trim("#docTranslationLink($supportedLocale $action $fragment)") +
-                  "</span>"
-                ))
-              #end
-            #end
-            <dd data-key="translations">$stringtool.join($output, ', ')</dd>
+        #foreach ($extension in $sortedExtensions)
+          #if ($extension.order < 1000)
```

Comment:
You're using 1000 as a limit number between static UIX and dynamic ones? Or I misunderstood something? 
Anyway that doesn't sound good: the order number should only be used for sorting the UIX, so a dynamic extension could be added in the middle of the static ones and vice-versa.

Reply:
I think you misunderstood something. The value 1000 is the limit between the left and the right column. All UIX (static and dynamic) with order less than 1000 are displayed in the left column and all UIX with a number equal or greater than 1000 are displayed in the right column. The UIX in both columns are sorted according to the order parameter. It is the freedom of the skin to divide the items into more or less columns, that's why the UIXP doesn't have an explicit `column` parameter and instead partitions by order. I should add a comment, though I thought this would also be part of the UIXP documentation.

Reply:
I have added a comment to the source to explain the rationale for splitting by `order` parameter.

Reply:
I will define a constant for the number. Just for reference, this splitting into columns has also been discussed in the [forum](https://forum.xwiki.org/t/new-user-interface-extension-point-uixp-for-page-information/9470/2) where I first proposed using a column parameter.